### DISCUSSION TITLE
Need information/advice about Mirth configuration for my first live channel

### DISCUSSION THREAD

**Discussion by TexasGaidheal** [nextgenhealthcare/connect#5166] [Q&A]
After a long learning process, with much useful assistance from the community of Mirth users, I am now very close to the point of deploying my first live production Mirth channel. I am starting with a channel that receives HL7 ORU messages with a TCP listener, converts the information into human-readable form, and then uses a file writer destination to save the results. Message volume is low - only about 50 messages arrive in a day. Eventually, I expect to have about 5 more similar channels that will run on the same system.

I need some information about best practices/configurations; in particular, I need advice/information about the log level settings that I should set in the Mirth Connect Server Manager and the data pruning settings that I should set in the Mirth Connect Administrator.

The hardware that will be running my first channel is not at the leading edge of technology - it is an HP Compaq 4300 with an Intel core i5 2.9 GHz processor and 12 Gb of memory. The OS is Windows 10 21H2.

There is only one disk drive, a 465 Gb drive with 232 Gb of free space.

Mirth is v4.0.0; Java is v17, database is PostgreSQL 14.

**Comment by jonbartels** [comment]
A good background on the pruner is this video from Zen - https://www.youtube.com/watch?v=oc-tLvQ2qkA
